DscResources/MSFT_xMySqlDatabase/MSFT_xMySqlDatabase.schema.mof

```diff
@@ -1,9 +1,10 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlDatabase")] 
 class MSFT_xMySqlDatabase : OMI_BaseResource
 {
-    [Key, Description("Name of the database.")] String Name;
-    [Required, Description("Should the database be present or absent."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
-    [Required, Description("The root credential that is used to install MySql server."), EmbeddedInstance("MSFT_Credential")] String ConnectionCredential;
+    [Key, Description("Name of the database.")] String DatabaseName;
```

Comment:
It seemed clearer to use DatabaseName.  It also makes it DatabaseName throughout the xMySql module.  Since it didn't work the way it was written, I didn't worry about breaking existing configurations.